formwidgets/revisionhistory/partials/_revisionhistory.htm

```diff
@@ -88,6 +123,11 @@ <h4 class="modal-title">
                                             data-request-success="$(this).closest('.modal').modal('hide');">
                                         <?= e(trans('samuell.revisions::lang.revision.restore_all')) ?>
                                     </button>
+                                    <button type="submit" class="btn btn-danger" data-request="onDeleteRevisionById"
```

Comment:
The added button pushes the cancel button to a new line for the at least the dutch language. Increasing the width of the model could solve this.
![image](https://user-images.githubusercontent.com/20305359/95579696-61305000-0a36-11eb-8cac-e9f1678ba3e7.png)

Reply:
> The added button pushes the cancel button to a new line for the at least the dutch language. Increasing the width of the model could solve this.
> ![image](https://user-images.githubusercontent.com/20305359/95579696-61305000-0a36-11eb-8cac-e9f1678ba3e7.png)

![image](https://user-images.githubusercontent.com/49227960/95836418-b1adf300-0d3f-11eb-86a9-053230f61b5d.png)

what about this? 
i add <div><br></div>

Reply:
![image](https://user-images.githubusercontent.com/49227960/95837119-84157980-0d40-11eb-9bc4-a6417f99fef1.png)

German words are bigger.

width of the model: October habe some ajax popup style for width like this:

`data-size="large" - Popup size, available sizes: giant, huge, large, small, tiny`

https://octobercms.com/docs/ui/popup